src/overlays/actors/ovl_Bg_Mori_Elevator/z_bg_mori_elevator.h

```diff
@@ -6,11 +6,19 @@
 
 struct BgMoriElevator;
 
+typedef void (*BgMoriElevatorActionFunc)(struct BgMoriElevator*, struct GlobalCtx*);
+
 typedef struct BgMoriElevator {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x28];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ BgMoriElevatorActionFunc actionFunc;
+    /* 0x0168 */ f32 targetYPos;
+    /* 0x016C */ s32 unk_16C;
+    /* 0x0170 */ u8 isRiding;
+    /* 0x0171 */ s8 objectIndex;
+    /* 0x0172 */ s16 unk_172;
+
 } BgMoriElevator; // size = 0x0174
 
 extern const ActorInit Bg_Mori_Elevator_InitVars;
 
-#endif
+#endif
```

Comment:
New line at end of file (also please remove that test file I asked you to add earlier :P)